package.json

```diff
@@ -150,7 +151,9 @@
     "cover:integration": "node --max-old-space-size=4096 node_modules/jest-cli/bin/jest --testMatch \"<rootDir>/test/*.test.js\" --coverage",
     "cover:unit": "node --max-old-space-size=4096 node_modules/jest-cli/bin/jest --testMatch \"<rootDir>/test/*.unittest.js\" --coverage",
     "cover:types": "node node_modules/tooling/type-coverage",
-    "cover:report": "istanbul report"
+    "cover:report": "istanbul report",
+    "postinstall": "yarn-deduplicate --fail --list -s fewer yarn.lock || { echo \"You must run yarn-deduplicate\"; exit 1; }",
```

Comment:
Why ask to run `yarn-deduplicate`? Let's run.

Reply:
After running it, you need to run yarn install again, that can cause another postinstall run

Reply:
My personal preferences to not automate is that far, because I fell it might break somehow

Reply:
I changed it to run it directly. It won't cause a loop as install is only run when deps are deduplicated and this happens only once.

I also changed it to not run in CI on postinstall, as this would hide non-deduplicated lockfile for linting on CI

Reply:
Sure,
I will also advice to use `yarn install --frozen-lock` and not just `yarn install`